test/java/ch/interlis/ili2c/Interlis24/GenericValueRanges24Test.java

```diff
@@ -0,0 +1,87 @@
+package ch.interlis.ili2c.Interlis24;
+
+import ch.ehi.basics.logging.EhiLogger;
+import ch.interlis.ili2c.CompilerTestHelper;
+import ch.interlis.ili2c.Ili2c;
+import ch.interlis.ili2c.Ili2cFailure;
+import ch.interlis.ili2c.LogCollector;
+import ch.interlis.ili2c.config.Configuration;
+import ch.interlis.ili2c.config.FileEntry;
+import ch.interlis.ili2c.config.FileEntryKind;
+import ch.interlis.ili2c.metamodel.TransferDescription;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class GenericValueRanges24Test {
+
+    private static final String TEST_OUT = "test/data/ili24/genericValueRanges/";
+
+    @Test
+    public void contextInSameModel() {
+        TransferDescription td = CompilerTestHelper.getTransferDescription(TEST_OUT + "contextInSameModel.ili");
+        assertNotNull(td);
+    }
+
+    @Test
+    public void contextInDifferentModel() {
+        TransferDescription td = CompilerTestHelper.getTransferDescription(TEST_OUT + "contextInDifferentModel.ili");
+        assertNotNull(td);
+    }
+
+    @Test
+    public void invalidDimensions() {
+        LogCollector errs = new LogCollector();
+        EhiLogger.getInstance().addListener(errs);
+
+        Configuration ili2cConfig=new Configuration();
+        FileEntry fileEntry=new FileEntry(TEST_OUT + "invalidDimension_fail.ili", FileEntryKind.ILIMODELFILE);
+        ili2cConfig.addFileEntry(fileEntry);
+        try {
+            Ili2c.runCompiler(ili2cConfig);
+        } catch (Ili2cFailure e) {
+        }
```

Comment:
REQ: Diese Zeilen in `CompilerTestHelper` verpacken. z.B. in `GetCompileErrors(string)`. Da könnten wir auch asserten, dass der Compiler eine Exception geworfen hat. 
```
try { 
  ili2c.runCompiler(..); 
  assert.Fail();
} catch (Ili2cFailure){ ... }
```